linux_os/guide/system/permissions/restrictions/enable_nx/bios_enable_execution_restrictions/tests/wrong_value.fail.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# platform = multi_platform_rhel,multi_platform_fedora
+#
+# remediation = none
+
+> /tmp/cpuinfo
+> /tmp/cmdline
+
+if [ -e /tmp/cpuinfo ] && [ -e /proc/cpuinfo ]; then
+cp /proc/cpuinfo /tmp/cpuinfo
+if grep -Fxq '^flags.*:.*/& nx' /tmp/cpuinfo ; then
```

Comment:
This `if` is not necessary since the `flags` field is always present in the file /proc/cpuinfo:
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/arch/x86/include/asm/cpufeature.h

Also, this filter will always return false. I would suggest to remove it.

Reply:
It is to make sure just in case, either someone else had the same destructive idea as us or there has been any strange changes.